{
  "name": "custom-error-generator",
  "version": "7.0.0",
  "description": "Custom errors and exceptions in Node.js",
  "main": "error.js",
  "scripts": {
    "test": "make test"
  },
  "engines": [
    "node >=0.6.0"
  ],
  "config": {
    "blanket": {
      "pattern": "/error.js",
      "data-cover-never": "node_modules"
    },
    "travis-cov": {
      "threshold": 80
    }
  },
  "author": {
    "name": "Jesse Proulx",
    "email": "github@jproulx.net"
  },
  "license": "ISC",
  "devDependencies": {
    "mocha": "",
    "should": "",
    "blanket": "",
    "mocha-lcov-reporter": "",
    "jshint": "",
    "travis-cov": "",
    "codeclimate-test-reporter": "",
    "coveralls": ""
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/jproulx/node-custom-error.git"
  },
  "keywords": [
    "error",
    "errors",
    "error factory",
    "custom error"
  ],
  "bugs": {
    "url": "https://github.com/jproulx/node-custom-error/issues"
  },
  "homepage": "https://github.com/jproulx/node-custom-error",
  "_id": "custom-error-generator@7.0.0",
  "dist": {
    "shasum": "b16f8d09636327f69ae80a1043766f30f4506408",
    "tarball": "http://registry.npmjs.org/custom-error-generator/-/custom-error-generator-7.0.0.tgz"
  },
  "_from": "custom-error-generator@^7.0.0",
  "_npmVersion": "1.4.3",
  "_npmUser": {
    "name": "jproulx",
    "email": "github@jproulx.net"
  },
  "maintainers": [
    {
      "name": "jproulx",
      "email": "github@jproulx.net"
    }
  ],
  "directories": {},
  "_shasum": "b16f8d09636327f69ae80a1043766f30f4506408",
  "_resolved": "https://registry.npmjs.org/custom-error-generator/-/custom-error-generator-7.0.0.tgz"
}
